### PR TITLE
woeusb: fix version, dependencies, split gui

### PR DESCRIPTION
**Pull request by sgn**
- fix version
- add bash as dependencies, the script is written in bash not sh
- man 1 woeusbgui said they recommend using non-gui version.
  Split them out for reduced dependencies.

### CODE REVIEW

**Comment by bra1nwave** [comment]
Why create a separate template (`woeusb`) and not just fixing the existing one (`WoeUSB`)?

**Comment by sgn** [comment]
- binary is all lowercase
- who installed WoeUSB will still get full version with gui, until they want to remove it and install woeusb.

Anyway, you're this package maintainer, it's your call.

I don't want to pull wxWidget dependencies.


On 12 April 2020 15:14:16 GMT+07:00, Daniel Boguslawski <notifications@github.com> wrote:
>Why create a separate template (`woeusb`) and not just fixing the
>existing one (`WoeUSB`)?
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/void-linux/void-packages/pull/20910#issuecomment-612580261

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by bra1nwave** [comment]
I think we should just change the existing one. There's no need to create a separate template IMHO

File: srcpkgs/woeusb/template [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+maintainer="bra1nwave <bra1nwave@protonmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/slacka/WoeUSB"
+distfiles="https://github.com/slacka/WoeUSB/archive/v${version}.tar.gz"
+checksum=0cab88a1113506f39d2f1c19532b2cd8d968c6a9f59129953c000e29e73f3d4f
+
+post_patch() {
+	vsed -i -e "s/@@WOEUSB_VERSION@@/$version/g" \
+		configure.ac \
+		src/woeusb src/woeusb.1 src/woeusbgui.1
```
Comment (bra1nwave) [inline srcpkgs/woeusb/template:22]: You can put this in `pre_configure()`

File: srcpkgs/woeusb/template [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+}
+
+pre_configure() {
+	autoreconf -fi
+}
+
+woeusbgui_package() {
```
Comment (bra1nwave) [inline srcpkgs/woeusb/template:29]: Please change it to `woeusb-gui`

File: srcpkgs/woeusb/template [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+	archs='*'
+	depends="${sourcepkg}-${version}_${revision}"
+	short_desc+=" - GUI wrapper"
+	pkg_install() {
+		vmove usr/bin/woeusbgui
+		vmove usr/share/applications
+		vmove usr/share/man/man1/woeusbgui*
```
Comment (bra1nwave) [inline srcpkgs/woeusb/template:36]: Put `usr/share/man/man1/woeusbgui*` in qoutation marks

File: srcpkgs/woeusb/template [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+revision=2
+wrksrc="WoeUSB-$version"
+archs=noarch
+build_style=gnu-configure
+configure_args="--with-wx-config=wx-config-gtk3"
+hostmakedepends="automake gettext libtool"
+makedepends="wxWidgets-gtk3-devel"
+depends="bash grub ntfs-3g parted wget"
+short_desc="Create a Windows USB stick installer from a real Windows DVD or image"
+maintainer="bra1nwave <bra1nwave@protonmail.com>"
```
Comment (bra1nwave) [inline srcpkgs/woeusb/template:13]: Also if you want you can take maintership for this package

**Comment by Chocimier** [comment]
Please keep main package WoeUSB, and create WoeUSB-cli or similar with `vmove /`.

**Comment by sgn** [comment]
On 2020-04-14 00:23:54-0700, Piotr <notifications@github.com> wrote:
> Please keep main package WoeUSB, and create WoeUSB-cli or similar with `vmove /`.

This is the best approach, IMHO.

-- 
Danh

File: srcpkgs/woeusb/template (hunk 1 above)
Comment (sgn) [reply srcpkgs/woeusb/template]: It's technically patching steps, I would like to have something to inspect after patching.

File: srcpkgs/woeusb/template (hunk 2 above)
Comment (sgn) [reply srcpkgs/woeusb/template]: we take different approach

File: srcpkgs/woeusb/template (hunk 4 above)
Comment (sgn) [reply srcpkgs/woeusb/template]: Nope, I happened to use this package for burning a Windows USB stick for boot.
Not really interested to follow their development.

**Comment by xtraeme** [comment]
Err wrong button heh

**Pull request merged**